src/shilpa/coding_exam_6/CharHavingMaxFreq.java

```diff
@@ -0,0 +1,44 @@
+/*Test -6 : [Complexity : Easy]
+Find a Character having a maximum frequency.
+
+input : globant india is hiring
+
+output : i*/
+package shilpa.coding_exam_6;
+
+public class CharHavingMaxFreq {
+	String getCharHavingMaxFreq(String str) {
+		String[] input = str.split("");
+		int cnt = 1;
+		int max = 0;
+		String maxstr = "";
+		for (int index1 = 0; index1 < input.length; index1++) {
```

Comment:
Take review of below code:
Komal - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/1767/files